chemicaltoolbox/datatypes/molecules.py

```diff
@@ -449,6 +449,33 @@ def set_peek( self, dataset, is_multi_byte=False ):
             dataset.peek = 'file does not exist'
             dataset.blurb = 'file purged from disk'
 
+class PDBQT(GenericMolFile):
+    """
+    Autodock Format
+    The PDBQT format stores the atomic coordinates, partial charges and AutoDock atom types,
+    for both the receptor and the ligand.
+    http://autodock.scripps.edu/faqs-help/faq/what-is-the-format-of-a-pdbqt-file
+    """
+    file_ext = "pdbqt"
+
+    def sniff(self, filename):
+        if count_special_lines('COMPND', filename) > 0:

```

Comment:
This datatype needs to go here: https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/datatypes/molecules.py
Galaxy now supports all chemical datatypes natively!! :)
I can do this is you like.